registries/_format/decimal.md

```diff
@@ -0,0 +1,21 @@
+---
+owner: baywet
+issue:
+description: A fixed point decimal number as defined by ISO/IEC 9075-2 2016 12 15
```

Comment:
Why the restriction to fixed-point decimal numbers?

How would I express that it is a [DECFLOAT](https://en.wikipedia.org/wiki/Decimal_floating_point)?

Reply:
I got the fixed point part from those two comments
https://github.com/OAI/OpenAPI-Specification/issues/845#issuecomment-378139730
https://github.com/OAI/OpenAPI-Specification/issues/845#issuecomment-392756538

Reply:
The first comment shows a table whose line for `decimal` references issue https://github.com/OAI/OpenAPI-Specification/issues/889, which in turn references Python and PostgreSQL:

> For money params we need special type, in Python, for example, it named [decimal](https://docs.python.org/3/library/decimal.html), PostgreSQL has [numeric](https://www.postgresql.org/docs/9.6/static/datatype-numeric.html#DATATYPE-NUMERIC-DECIMAL) type.

The first source (Python) explicitly includes decimal floating-point:

> [decimal](https://docs.python.org/3/library/decimal.html#module-decimal) — Decimal fixed point and floating point arithmetic

Reply:
```suggestion
description: A fixed or floating point decimal number as defined by ISO/IEC 9075-2 2016 12 15
```

Thanks for the additional context, what do you think of this?

Reply:
Perfect, ISO/IEC 9075-2 also defines `DECFLOAT`